drivers/i2c/i2c_mchp_xec.c

```diff
@@ -16,11 +16,10 @@
 #define SPEED_400KHZ_BUS    1
 #define SPEED_1MHZ_BUS      2
 
-#define WAIT_INTERVAL       5
-/* 250 us */
-#define TIMEOUT       100
-/* 25 us */
-#define MAX_CLK_STRETCHING  5
+#define EC_OWN_I2C_ADDR		0x7F
```

Comment:
Generic name required (not EC related)

Reply:
OWN address is one of the two the follower addresses this controller can respond to.  Should these values be obtained from device tree?